columnphysics/icepack_therm_itd.F90

```diff
@@ -306,11 +306,24 @@ subroutine linear_itd (ncat,        hin_max,     &
 
          if (hicen_init(n)   > puny .and. &
              hicen_init(n+1) > puny) then
-             ! interpolate between adjacent category growth rates
-             slope = (dhicen(n+1) - dhicen(n)) / &
+
+            if (hicen_init(n+1) - hicen_init(n))>0
```

Comment:
line 310 is missing 'then'.
Should this be >puny, or is >0 sufficient to catch the error?  Roundoff...

Reply:
Yes, I'll fix this up.  Just a draft at this stage in case everyone wanted to go in another direction.